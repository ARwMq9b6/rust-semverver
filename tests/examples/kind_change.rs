pub mod old {
    pub struct Abc;
}

pub mod new {
    pub enum Abc {

    }
}
