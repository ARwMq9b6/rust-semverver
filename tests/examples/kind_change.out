version bump: 1.0.0 -> (Breaking) -> 2.0.0
  Removal: Abc
  Abc -
    KindDifference
